Library/Homebrew/cask/dsl.rb

```diff
@@ -521,6 +527,26 @@ def livecheckable?
       @livecheck_defined == true
     end
 
+    def no_autobump!(because:)
+      raise ArgumentError, "`because` argument must be a string!" unless because.is_a?(String)
+
+      if !@cask.allow_reassignment && @no_autobump_defined
+        raise CaskInvalidError.new(cask, "'no_autobump_defined' stanza may only appear once.")
+      end
+
+      @no_autobump_defined = true
+      @no_autobump_msg = because
+      @autobump = false
+    end
+
+    def autobump?
+      @autobump == true
```

Comment:
```suggestion
      tap&.official? && @autobump == true
```
or similar. 

May be we just change whether it defaults to true/false depending on official tap or not.

Reply:
Not sure if it's needed, I do not see any use of autobump list outside of `brew bump` and `brew livecheck` commands and they already have limit autobump usage to the official taps

Reply:
@botantony We should assume people will try to use all DSLs outside of the official tap, even if they don't make sense, unless we prevent it. Could just do a `raise ArgumentError, "Only for official taps!" unless tap.official?` or similar?